docs/reference/commandline/index.md

```diff
@@ -62,20 +62,20 @@ read the [`dockerd`](dockerd.md) reference page.
 | [events](events.md) | Get real time events from the server                   |
 | [exec](exec.md) | Run a command in a running container                       |
 | [export](export.md) | Export a container's filesystem as a tar archive       |
-| [kill](kill.md) | Kill a running container                                   |
+| [kill](kill.md) | Kill one or more running containers                        |
```

Comment:
@mstanleyjones wdyt? Do you have suggestions? I find the "one or more" everywhere a bit awkward.